build/docker/centos6/devel/Dockerfile

```diff
@@ -5,28 +5,37 @@ FROM ${REPOSITORY}:${VERSION}
 # add vscode server
 RUN yum repolist && \
     yum -y install \
+        bash-completion \
+        byobu \
+        cgdb \
+        emacs-nox \
+        jq \
+        the_silver_searcher \
         tmux \
         tree \
-        emacs-nox \
         vim \
-        bash-completion \
-        jq \
-        cgdb && \
+        zsh && \
     yum clean all && \
     rm -rf /var/cache/yum
 
 WORKDIR /tmp
 RUN source /opt/rh/devtoolset-8/enable && \
     source /opt/rh/rh-python36/enable && \
     pip3 install \
+        lxml \
+        psutil \
         python-dateutil \
-        subprocess32 \
-        psutil && \
+        subprocess32 && \
     mkdir fdb-joshua && \
     cd fdb-joshua && \
     git clone --branch code_pipeline https://github.com/FoundationDB/fdb-joshua . && \
     pip3 install /tmp/fdb-joshua && \
     cd /tmp && \
+    curl -Ls https://amazon-eks.s3.us-west-2.amazonaws.com/1.18.9/2020-11-02/bin/linux/amd64/kubectl -o /usr/local/bin/kubectl && \
```

Comment:
can you please use a checksum to verify that these didn't get corrupted? Otherwise security will be sad :)